Samples/WindowsML/cs-winui/MainWindow.xaml.cs

```diff
@@ -135,7 +135,8 @@ private async Task LoadModelAsync()
         private async Task LoadAndRunDemoAsync()
         {
             // Get application directory
-            var appDirectory = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+            var appDirectory = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)
```

Comment:
Consider if this works for your scenarios or not: https://learn.microsoft.com/en-us/dotnet/api/system.appcontext.basedirectory?view=net-9.0